modules/acm/README.md

```diff
@@ -18,23 +18,39 @@ Other clusters should disable feature activation by setting `enable_fleet_featur
 There is a [full example](../../examples/simple_zonal_with_acm) provided. Simple usage is as follows:
 
 ```tf
+provider "kubernetes" {
+  host                   = "https://${module.gke.endpoint}"
+  token                  = data.google_client_config.default.access_token
+  cluster_ca_certificate = base64decode(module.gke.ca_certificate)
+}
+
```

Comment:
Would it be preferable to also have ```data "google_client_config" "default" {}``` here?

Reply:
Missed that, thanks!